core/src/main/res/layout/dialog_add_note.xml

```diff
@@ -22,22 +22,20 @@
 
       <TextView
         android:id="@+id/add_note_text_view"
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:background="@android:color/transparent"
         android:hint="@string/wiki_article_title"
-        android:maxLines="1"
+        android:maxLines="3"
         android:paddingStart="20dp"
         android:paddingTop="10dp"
         android:paddingEnd="20dp"
-        android:textAppearance="@style/TextAppearance.MaterialComponents.Headline5" />
+        android:textAppearance="@style/TextAppearance.MaterialComponents.Subtitle1" />
```

Comment:
`?textAppearanceSubtitle1` use attributes and preferably our attributes where possible

Reply:
Sorry, I missed that.

Reply:
@macgills done.